variables.tf

```diff
@@ -67,3 +67,11 @@ variable "labels" {
   type        = map(string)
   default     = {}
 }
+variable "lifecycle_rules" {
+  type = set(object({
+    action = map(string)
+    condition = map(string)
+  }))
+  description = "List of lifecycle rules to configure. Format is the same as described in provider documentation https://www.terraform.io/docs/providers/google/r/storage_bucket.html#lifecycle_rule except condition.matches_storage_class should be a comma delimited string."
+  default     = []
+}
```

Comment:
The pattern we followed everywhere is
- description
- then type
- then default

Also, consider running `terraform fmt -recursive` to format all lines of all files correctly 🙂